### DISCUSSION TITLE
How to render different content on server and client

### DISCUSSION THREAD

**Discussion by milelazar** [nuxt/nuxt#25066] [Questions] [answered]
I have one very large and complex menu. There is no need to render all that on server so to improve server performance my idea is to render simplified ssr version (just regular ul list with links) and complex version only on the client.

How can i do that and is that even advisable? There is `<ClientOnly>` component for client only content. It would be nice to have something like `<ServerOnly>`

**Comment by TheAlexLichter** [comment, marked as answer]
ClientOnly + `fallback` slot should work fine 😋

**Reply by milelazar** [reply]
Ah, didn't think of that. Works perfectly. Thank you
